tests/cmake_integration/generated_global/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Distributed under the OpenDDS License. See accompanying LICENSE
+# file or http://www.opendds.org/license.html for details.
+#
+# This file is setup the way it is for testing purposes. For a cleaner example
+# see DevGuideExamples/DCPS/Messenger/CMakeLists.txt
+
+project(opendds_generated_property CXX)
+cmake_minimum_required(VERSION 3.8.2)
+
+find_package(OpenDDS REQUIRED)
+
+set(CMAKE_CXX_COMPILER ${OPENDDS_COMPILER})
+
+if (NOT OPENDDS_OWNERSHIP_PROFILE)
+  message(FATAL_ERROR "These tests require OpenDDS to be built with Ownership Profile enabled")
+endif()
+
+set(target_prefix "opendds_cmake_generated_global_")
+set(publisher "${target_prefix}publisher")
+set(messenger "${target_prefix}messenger")
+
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/../idl)
+link_directories(${CMAKE_CURRENT_BINARY_DIR}/../idl)
+
```

Comment:
```suggestion
```

I don't think these are doing anything. Also these types of commands aren't considered good practice in CMake and the documentation warns against using them.